packages/sirius-web/backend/sirius-web-domain/src/main/java/org/eclipse/sirius/web/domain/boundedcontexts/semanticdata/services/SemanticDataUpdateService.java

```diff
@@ -45,4 +46,12 @@ public void updateDocuments(ICause cause, AggregateReference<SemanticData, UUID>
             this.semanticDataRepository.save(semanticData);
         });
     }
+
+    @Override
+    public void addDependencies(ICause cause, AggregateReference<SemanticData, UUID> semanticDataId, List<AggregateReference<SemanticData, UUID>> dependencySemanticDataIds) {
```

Comment:
I don't really know if the list of dependencies should be a `List<UUID>` or a `List<AggregateReference<SemanticData, UUID>>`

Reply:
`List<AggregateReference<SemanticData, UUID>>` is fine